Source/FileLiberator/_InternalsVisible.cs

```diff
@@ -1 +0,0 @@
-﻿[assembly: System.Runtime.CompilerServices.InternalsVisibleTo(nameof(FileLiberator) + ".Tests")]
```

Comment:
Are tests no longer hitting protected items?

Reply:
Nope. IDK when that changed, but it did.